GOVERNANCE.md

```diff
@@ -0,0 +1,340 @@
+and other project activities as peers with all other Contributors and the Community. 
+In these everyday activities, Council Members do not have any special power or privilege through 
+their membership on the Council. However, it is expected that because of the quality and quantity of
+their contributions and their expert knowledge of the Project Software and Services that Council 
+Members will provide useful guidance, both technical and in terms of project direction, 
+to potentially less experienced contributors.
+
+The Council will have between 4 and 7 members. No more than 2 Council Members can report to one 
+person or company (including Institutional Partners) through employment or contracting work 
+(including the reportee, i.e. the reportee + 1 is the max).
+
+#####  Council Responsibilities
+
+Council Members will have the responsibility of
+
+* Removing members, including Council Members, if they are in violation of the Code of Conduct 
+or don't comply with this governance document.
+* Making decisions when regular community discussion does not produce consensus on an issue
+in a reasonable time frame. See [Council Decision Making Process](#council-decision-making-process) 
+page for more details.
+
+The Council may choose to delegate these responsibilities to sub-committees. 
+If so, Council members must update this document to make the delegation clear.
+
+Individual Council Members do not have the power to unilaterally wield these responsibilities. 
+The Council as a whole must jointly make these decisions. 
+In other words, Council Members are first and foremost Core Contributors, but only when needed they 
+can collectively make decisions for the health of the project.
+
+##### Length of Tenure and Reverification
+
+<!-- NOTE Everything here is open to discussion -->
+
+* Council members term limits are 4 years, after which point their seat will come up for reelection.
+* Each year on April 7th council members will be asked to restate their commitment to being on the 
+Council.
+* Attempts should be made to reach every council member over at least 2 communication media. 
+For example: email, Slack, phone, or GitHub.
+* If a Council Member does not restate their commitment their seat will be vacated.
+* Inactivity can be determined by lack of substantial contribution, including votes on council, 
+code or discussion contributions, contributions in the community or otherwise.
+* In the event of a vacancy in the council, an [election](#council-selection-process) will be held 
+to fill the position.
+* There is no limit on the number of terms a Council Member can serve
+
+##### Current Council members
+
+The current Council members are:
+
+* Gabriel Stechschulte ([@gstechschulte](https://github.com/gstechschulte))
+* Osvaldo Martin ([@aloctavodia](https://github.com/aloctavodia))
+* Ravin Kumar ([@canyon289](https://github.com/canyon289))
```

Comment:
```suggestion
```

Removing myself so you three have less voting overhead and with votes for people that are more familiar with the package